### DISCUSSION TITLE
icons are cut in half.

### DISCUSSION THREAD

**Discussion by kbc8894** [blinksh/blink#1873] [Contributors Wanted]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

- Blink Version: v16.1.11.798
- iPadOS Version: 17.0

### Describe the bug

Hello!

In Blink, when using mosh, everything appears normal, but when using ssh, only half icon is visible.
Additionally, in neovim, the icons are cut in half.

Help me.
Thanks

- using ssh
<img width="618" alt="image" src="https://github.com/blinksh/blink/assets/3617939/ce6db3de-ac45-466e-ab1e-9464df604fde">

- using mosh 
<img width="547" alt="image" src="https://github.com/blinksh/blink/assets/3617939/4b7c7b23-7321-4e73-86ec-2884fda5e7a4">

- using mosh & neovim
<img width="1366" alt="image" src="https://github.com/blinksh/blink/assets/3617939/4f98d74a-ebd3-4b20-a417-a5d016a403ed">

**Comment by carloscabanero** [comment]
Thanks for the heads up. Will take a look at this. Can you send us the font you are using and the shell prompt?

**Comment by kbc8894** [comment]
> Thanks for the heads up. Will take a look at this. Can you send us the font you are using and the shell prompt?

@carloscabanero
Above screenshots are using [MesloLGS Nerd Font](https://github.com/kbc8894/blinksh-fonts/blob/master/MesloLGS%20Nerd%20Font.css) (generated by [link](https://github.com/blinksh/fonts/blob/master/generate-nerd-fonts.py))
And,
[Ubuntu Mono Nerd Font](https://github.com/blinksh/fonts/blob/master/Ubuntu%20Mono%20Nerd%20Font.css) is same.

**Reply by carloscabanero** [reply]
We get this one quite often, so I created https://github.com/blinksh/blink/issues/1874.

The problem is our fonts fell behind the patches. You need to patch the new ones yourself and use proper Mono, not regular or proportional.

If someone wants to take on a small project like this, please let me know. It would be great to give a bump to our repository. We can do it easily with a Python script.

**Reply by carloscabanero** [reply]
Just a heads up. This should now be fully fixed since a few months ago and it doesn't require any special font treatment.
